packages/custom-functions-metadata-plugin/src/loader.ts

```diff
@@ -0,0 +1,11 @@
+import { getOptions } from "loader-utils";
+import { IAssociate } from "custom-functions-metadata";
+
+module.exports = function addFunctionAssociations(this: any, source: string): string {
+    const input = getOptions(this).input as string;
+    if (!global.generateResults || !global.generateResults[input]) return source;
```

Comment:
Let's indent this to be like:
```
if (!global.generateResults || !global.generateResults[input]) {
    return source;
}
```
To be more consistent with the rest of the codebase.